types/stylelint/index.d.ts

```diff
@@ -84,6 +86,22 @@ declare namespace stylelint {
 		severity?: Severity;
 	};
 
+	type LanguageOptions = {
```

Comment:
question

Since you patch `css-tree`, can't you use that type instead?
e.g.
```ts
import type { fork } from 'foo';

type LanguageOptions = {
  syntax: Parameters<fork>[0];
};
```

Reply:
In that case, the stylelint side should also be `atrules` instead of `atRules`.

Reply:
Since you are already patching it, does it matter?
i.e. will the patch stay?

Reply:
If the options become `atrules`, would that be okay? 
I don't have a strong preference, so I'm okay either way.
https://github.com/stylelint/stylelint/blob/8d8e75b54770da0711977641a50888b38dab2e5a/docs/user-guide/configure.md?plain=1#L314

Reply:
You would need to remap it based on the other object and use [`as`](https://www.typescriptlang.org/docs/handbook/2/mapped-types.html#key-remapping-via-as).
This sounds too complicated.
i.e. not worth the effort
I wanted not to have to maintain our own type just for a camelcase difference.

Ill ask another maintainer just in case.
@ybiquitous @jeddy3 @romainmenke 
Should we base our type from theirs or not?

Reply:
What I was trying to get at is, whether we create that type from theirs and rename `atrules` to `atRules` using `as` or just maintain our own just due to that one difference.

Reply:
If the types don’t match exactly, it might be better to have our own format. 
Using `as` to rename and extend could make things harder to understand.

Reply:
Alright, we can reconsider this decision in the PR that will remove the patch.